.github/workflows/lint.yml

```diff
@@ -12,3 +12,5 @@ jobs:
             fetch-depth: 0
         - name: "lint: awesome-lint"
           uses: jthegedus/github-action-awesome-lint@v0.1.0 # https://github.com/jthegedus is part of Docsify core team, so we will use Tag versioning over commit_sha
+          env:
+            AWESOME_LINT_VERSION: "0.14.0" #0.15.0 Invalid URL: http:#showcase
```

Comment:
@jthegedus hi, Do you have time to review this PR?

Reply:
Can you elaborate? What is the invalid URL?

Reply:
https://github.com/docsifyjs/awesome-docsify#contents These links are parsed incorrectly.

Reply:
Not a complete link, but an anchor

Reply:
Okay, do we need the anchor? Can you raise an issue with the root tool https://github.com/sindresorhus/awesome-lint

Reply:
I didn't see this issue in a prev PR I just recently merged

Reply:
I think it was a weird runtime error as I reran your other PR's CI check and it didn't fail on that issue (it failed on other issues I have resolved in master so a rebase should fix)

Reply:
I think this should be reverted as I would like to keep up to date with the `latest` of `awesome-lint`

Reply:
done.